Counter/project.json

```diff
@@ -16,12 +16,23 @@
         "Microsoft.ServiceFabric": "5.0.135",
         "Microsoft.ServiceFabric.AspNetCore.Hosting": "1.0.0-*",
         "Microsoft.ServiceFabric.Services": "2.0.135"
+      },
+      "frameworkAssemblies": {
+        "System.Runtime": { "type": "build" },
```

Comment:
Is there any reason System.Runtime and System.ComponentModel need to be added here? Is it because of DNX? Building with DotNet CLI doesn't seem to require the framework assemblies.